ls1intum/Artemis
# Allow to set timeout for jenkins pipelines

Fixes #2434

### Checklist


### Motivation and Context
Timeout out removes stale builds so i don't have to to it manually

### Description


### Steps for Testing

Use TS2 for testing
1. Create a programming exercise
2. Go to Edit in Editor, change to the Test Repository and remove the @StrictTimeout annotations from SortingBehaviorTest.java
3. Participate in the exercise
4. Submit valid code with an endless loop
5. Check the build plan in Jenkins. It should be stopped after 30 mins.

I have no set up server. So I would be cool if one could help me test this or something similar.
